compiler/jsgen.nim

```diff
@@ -2496,21 +2496,19 @@ proc newModule(g: ModuleGraph; module: PSym): BModule =
   result.config = g.config
 
 proc genHeader(): Rope =
-  result = (
-    "/* Generated by the Nim Compiler v$1 */$n" &
-    "/*   (c) " & copyrightYear & " Andreas Rumpf */$n$n" &
-    "var framePtr = null;$n" &
-    "var excHandler = 0;$n" &
-    "var lastJSError = null;$n" &
-    "if (typeof Int8Array === 'undefined') Int8Array = Array;$n" &
-    "if (typeof Int16Array === 'undefined') Int16Array = Array;$n" &
-    "if (typeof Int32Array === 'undefined') Int32Array = Array;$n" &
-    "if (typeof Uint8Array === 'undefined') Uint8Array = Array;$n" &
-    "if (typeof Uint16Array === 'undefined') Uint16Array = Array;$n" &
-    "if (typeof Uint32Array === 'undefined') Uint32Array = Array;$n" &
-    "if (typeof Float32Array === 'undefined') Float32Array = Array;$n" &
-    "if (typeof Float64Array === 'undefined') Float64Array = Array;$n") %
-    [rope(VersionAsString)]
+  result = rope("""/* Generated by the Nim Compiler v$1 (c) $2 Andreas Rumpf */
```

Comment:
Since we're at it, I think this is problematic as I claim copyrights for the Nim compiler but not for the code it produces which is subject to the copyright's owner of the Nim code that was translated.

Reply:
What does it means?, should I just remove the comment?.

Reply:
Remove the "(c) $year Andreas Rumpf" part.

Reply:
Fixed.